sdk/python/kubeflow/training/api/training_client_test.py

```diff
@@ -17,11 +18,23 @@
 from kubernetes.client import V1ResourceRequirements
 import pytest
 
-LIST_RESPONSE = [{"metadata": {"name": "Dummy V1PodList"}}]
+LIST_RESPONSE = [
+    {"metadata": {"name": "Dummy V1PodList-1"}},
+    {"metadata": {"name": "Dummy V1PodList-2"}},
+]
```

Comment:
Why do you want to change it ?

Reply:
Just to test and present the possibility of returning more than one name (I can remove this case if you think its unnecessary)

Reply:
@andreyvelich is it fine by you to keep this case?

Reply:
I would suggest keep it as it is for old tests, since in that case we don't need to introduce SimpleNamespace from typing module.